bin/dump.sh

```diff
@@ -16,10 +24,10 @@ dump() {
   log "Dumping certificates"
   traefik-certs-dumper file \
     --version v2 \
-    --crt-name "cert" \
-    --crt-ext ".pem" \
-    --key-name "key" \
-    --key-ext ".pem" \
+    --crt-name ${certificate_file_name} \
+    --crt-ext ${certificate_file_ext} \
+    --key-name ${privatekey_file_name} \
+    --key-ext ${privatekey_file_ext} \
```

Comment:
```suggestion
    --crt-name "${certificate_file_name}" \
    --crt-ext "${certificate_file_ext}" \
    --key-name "${privatekey_file_name}" \
    --key-ext "${privatekey_file_ext}" \
```

Add quotes in order to prevent globbing. See also https://github.com/koalaman/shellcheck/wiki/SC2086.
https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
Thanks for the hint, as I am not a bash expert

Reply:
...and just added https://github.com/timonwong/vscode-shellcheck to VsCode 😉